forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/connector/support/FormAccessHandler.java

```diff
@@ -61,16 +74,15 @@ private String getDecoratedServerUrl(String url) {
     }
 
     private String getToken() {
-        VariableInstance accessToken = ProcessEngines.getDefaultProcessEngine().getRuntimeService().createVariableInstanceQuery().variableName(getTokenName()).singleResult();
-        if(accessToken != null) {
-            return String.valueOf(accessToken.getValue());
+        List<VariableInstance> accessTokens = ProcessEngines.getDefaultProcessEngine().getRuntimeService().createVariableInstanceQuery().variableName("formio_access_token").orderByActivityInstanceId().desc().list();
```

Comment:
This doesn't seems right for me (The order by ). I tried to reproduce the issue by executing query "SELECT v.* FROM public.act_ru_variable v JOIN act_ru_execution e
	on (v.proc_inst_id_ = e.proc_inst_id_)
	where name_='formio_access_token'
	order by e.ACT_INST_ID_ desc"
Let me know if this is the query it executed, else let us discuss the output for the order by.

Reply:
Please add clarity on what does not seem right and scenario where it is not successful, with actual and expected results. 
On evaluating the test result, kindly approve and merge or send back.